helm/nginx-ingress-controller-app/templates/controller-deployment.yaml

```diff
@@ -32,9 +32,8 @@ spec:
   minReadySeconds: {{ .Values.controller.minReadySeconds }}
   template:
     metadata:
-      annotations:
-        cluster-autoscaler.kubernetes.io/safe-to-evict: "true"
```

Comment:
would it make sense to add this annotation as default in `Values.controller.podAnnotations`?